spec/controllers/admin/rentals_controller_spec.rb

```diff
@@ -97,6 +97,40 @@
       end
     end
 
+    context "when there is large amount of blank remote accounts" do
+      let!(:blank_remote_accounts) { create_list(:remote_account, 23, account: account) }
```

Comment:
I think 2 or 3 blank remote accounts should be enough here.

Reply:
Yeah, using that many of them is bad idea, I wanted to show that we will list all blank accounts first + paginated non blank accounts(so that old behaviour works when new config is not used), though with current expectation that is definitely not clear.
I'll use smaller number for per page pagination option and modify expectation.